### PR TITLE
Project complete

### PR DESCRIPTION
**Pull request by Dachrono**
Mobile screen

Dropdown menu that contains the link to the Home or About page, this depends on what page you are.
You can close the menu with the icon (x) and open it with the menu icon.
The menu icon always appears in the screen while you scroll down.
Desktop screen:

The design for the menu change but it still has the links to the other page Home or About depending on what of them you are.
The logo has a link to the home page.
The nav bar always appears in the top of the screen while you scroll down.
The “Features Speakers” is created dynamically using JavaScript.
Media query for the desktop design

Deploy the project, the link is in the README in the section Live Demo

This branch count with merges from different branches (you can see them in the commits history), this is because I created different ones for:

Mobile home
Mobile About
Home desktop
About desktop
Mobile menu
Interactions
Deployment

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -48,7 +52,7 @@ <h3>"Hello! Pets lovers"</h3>
             <h1>Veterinary international meet alternative natural food</h1>
         </div>
         
-        <p>The meeting to present an alternative diet for our pets is held 
+        <p id="introDesc">The meeting to present an alternative diet for our pets is held 
             annually to present the advances in this field. A more balanced 
             and natural diet represents an improvement in the quality of life 
             and a reduction in visits to the clinic for our pets.</p>
```
Comment (abdallahmalima) [inline about.html:58]: - 🙏Please decrease the size of the box🎁 containing the description and change the background color of the box to white, matching the original design. This adjustment will help preserve the coolness🎉 of your design, similar to the original.

Your design:
image
Original design:
image

File: about.html [hunk 2]
```diff
@@ -16,17 +16,21 @@
...
     <div class="menuNav">
-        <nav id="navBar" class="navBar">     
-            <a href="about.html" class="about">About</a>
+
+        <button class="campaign">CC Campaign</button>
+
+        <nav id="navBar" class="navBar">   
+            <a href="index.html" id="home">Home</a>  
+            <a href="about.html" id="about">About</a>
             <a href="" class="program">Program</a>
```
Comment (abdallahmalima) [inline about.html:34]: - 🙏Please ensure that the correct background color is used in the 📱mobile size screen, as specified in the original design. This adjustment will enhance the appearance of your project.

Your design:
image
Original design:
image

**Review by abdallahmalima — CHANGES_REQUESTED** [review]
Hi @Dachrono , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
- "Acknowledgements" section✔
- Project deployed✔
- `Home` page and `About` page✔

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html [hunk 3]
```diff
@@ -16,17 +16,21 @@
 <body>
     <header>
         <ul>
-            <li><img src="pics/icon/facebook.png" alt="Facebook"></li>
-            <li><img src="pics/icon/twitter.png" alt="Twitter"></li>
+            <li><img id="face" src="pics/icon/facebook.png" alt="Facebook"></li>
+            <li><img id="twi" src="pics/icon/twitter.png" alt="Twitter"></li>
             <li>English</li>
             <li>My page</li>
             <li>Logout</li>
```
[the remark this answers is not in the document]
Comment (Dachrono) [reply about.html:23]: Solved, tky for you advise

File: index.html [hunk 4]
```diff
@@ -48,13 +52,13 @@ <h3>"Hello! Pets lovers"</h3>
             <h1>Veterinary international meet alternative natural food</h1>
         </div>
         
-        <p>Welcome fellow veterinarians to the presentation of an 
+        <p id="introDesc">Welcome fellow veterinarians to the presentation of an 
             alternative food for domestic pet lovers, a food that 
             will revolutionize the way we feed our furry ones, for 
             the first time in Mexico</p>
         
         <div class="time">
```
[the remark this answers is not in the document]
Comment (Dachrono) [reply index.html:60]: Solved, tky for you advise

File: about.html (hunk 1 above)
Comment (Dachrono) [reply about.html:58]: Solved, tky for you advise

File: about.html (hunk 2 above)
Comment (Dachrono) [reply about.html:34]: Solved, tky for you advise

File: about.html (hunk 3 above)
Comment (OyePriscilla) [inline about.html:23]: - [ ] Awesome implementing this in `index.html` file, kindly do the same in about.html file as well. **Note: The code reviewer actually pointed at about.html document.** 👍

File: about.html [hunk 5]
```diff
@@ -48,7 +52,7 @@ <h3>"Hello! Pets lovers"</h3>
             <h1>Veterinary international meet alternative natural food</h1>
         </div>
         
-        <p>The meeting to present an alternative diet for our pets is held 
+        <p id="introDesc">The meeting to present an alternative diet for our pets is held 
             annually to present the advances in this field. A more balanced 
             and natural diet represents an improvement in the quality of life
```
Comment (OyePriscilla) [inline about.html:57]: - [ ] Kindly change the background color to what we have in the design as  raised earlier, our ability to reproduce a given template perfectly shows how well we pay attention to details. 👍 

## Your Design

Untitled

## Figma Design

Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Dachrono 👏 👏 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so-close-so-close-seth-meyers

 ## Highlights
Implemented two html pages ✔️
Implement eventlistener using `js` ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 3 above)
Comment (Dachrono) [reply about.html:23]: 😸 solved, I forget the second page, tky for your advise

File: index.html [hunk 6]
```diff
@@ -16,17 +16,21 @@
...
+            <li><a href="#">My page</li></a>
+            <li><a href="#">Logout</li></a>
         </ul>
     </header>
 
     <div class="menuNav">
-        <nav id="navBar" class="navBar">     
-            <a href="about.html" class="about">About</a>
+
+        <button class="campaign">CC Campaign</button>
```
Comment (tufoinnkuo10) [inline index.html:29]: - [optional] nice work. please, for you to really align with the main design, the speaker section of the mobile version should be displaying two speakers. You can add the more button functionality if you want to show extra speakers. Nice job.

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:✔️✔️ 

Hi @Dachrono ✔️✔️ ✔️✔️ 

Your project is complete! Great job for taking time to make the changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request✔️✔️ 
- Behance design well followed ✔️✔️ 
- well deployed✔️✔️ 
- work well presented✔️✔️ 
- No linter errors✔️✔️ 

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

smile

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**